src/Libraries/RevitNodes/Elements/ModelCurve.cs

```diff
@@ -269,41 +262,8 @@ private static string resetSketchPlaneMethod(Autodesk.Revit.DB.CurveElement mc,
                 return (sp == null || mc.SketchPlane.Id == sp.Id) ? "" : sp.UniqueId;
             }
 
-            //do reset if method is available
-
-            bool foundMethod = false;
-
-            if (hasMethodResetSketchPlane)
-            {
-                Type CurveElementType = typeof(Autodesk.Revit.DB.CurveElement);
-                MethodInfo[] curveElementMethods = CurveElementType.GetMethods(System.Reflection.BindingFlags.Instance | System.Reflection.BindingFlags.Public);
-                System.String nameOfMethodSetCurve = "ResetSketchPlaneAndCurve";
-                System.String nameOfMethodSetCurveAlt = "SetSketchPlaneAndCurve";
-
-                foreach (MethodInfo m in curveElementMethods)
-                {
-                    if (m.Name == nameOfMethodSetCurve || m.Name == nameOfMethodSetCurveAlt)
-                    {
-                        object[] argsM = new object[2];
-                        sp = GetSketchPlaneFromCurve(c);
-                        argsM[0] = sp;
-                        argsM[1] = null;
```

Comment:
Here is the most strange place. It always fails with exception curve can't be null. I believe here should be not null, but something meaningful.